examples/dao/test/dao-app-flow.js

```diff
@@ -0,0 +1,457 @@
+const { getProgram, convert } = require('@algo-builder/algob');
+const { Runtime, AccountStore } = require('@algo-builder/runtime');
+const { types, parsing } = require('@algo-builder/web');
+const { assert } = require('chai');
+const { ProposalType, Vote } = require('../scripts/run/common/common');
+const {
+  now, getAddProposalTx, getDepositVoteTokenTx, getWithdrawVoteDepositTx,
+  getClearVoteRecordTx, getClearProposalTx
+} = require('../scripts/run/common/tx-params');
+
+const minBalance = 10e6; // 10 ALGO's
+const initialBalance = 200e6;
+
+/**
+ * Test for scripts flow. Steps:
+ * + create DAO (setup)
+ *    1. Create Gov Token (ASA)
+ *    2. Create DAO App
+ *    3. Compile & fund lsig's (deposit, daoFund, proposal)
+ *    4. Add depositLsig address to DAO app
+ * + add proposal
+ * + deposit vote tokens
+ * + register vote
+ * + execute proposal
+ * + withdraw vote deposit
+ * + clear vote record
+ * + clear proposal
+ * https://paper.dropbox.com/doc/Algo-DAO--BTR~tKj8P788NMZqnVfKwS7BAg-ncLdytuFa7EJrRerIASSl
+ */
+describe('DAO test', function () {
+  const master = new AccountStore(1000e6);
+  let creator = new AccountStore(initialBalance);
+  let proposer = new AccountStore(initialBalance);
+  let voterA = new AccountStore(initialBalance);
+  let voterB = new AccountStore(initialBalance);
+  let depositLsigAcc = new AccountStore(initialBalance); // runtime.account of depositLsig.address()
+  let daoFundLsigAcc = new AccountStore(initialBalance);
+  let proposalLsigAcc = new AccountStore(initialBalance);
+
+  let runtime;
+  let flags;
```

Comment:
what are the "flags"? Could you add a comment?

Reply:
yes